bin/indexer

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env php
 <?php
 require_once(getenv('COLLECTIVEACCESS_HOME') . '/setup.php');
-$sleep = getenv('WORKER_INDEXING_SLEEP_TIME') ?? 1;
```

Comment:
This will mean we can never set this value to anything that evaluates to false. I think that is fine, but might be a trick if someone expects setting it to zero to have an effect.